### ISSUE TITLE
Add a 'p' argument to HTTP requests for Mongo projection

### ISSUE THREAD

**Issue by TanukiSharp** [LartTyler/MHWDB-API#62]
As the `q` argument, it would be great to have a `p` argument to project result data.
The benefit is great for tool developers to pre-filter what they don't need instead of processing it locally, and for you to save bandwidth consumption (not sure how this is billed, but anyway it is always good to avoid wasting resources).

I tried to provide a `p` argument but this seemed to not work. Since it is not documented, I assume the feature does not exist yet.

Thanks for your consideration.

**Comment by LartTyler** [comment]
Before I finalize these changes, I'd love to get some feedback. I'm currently planning on implementing projection in the same way that Mongo does. For example:

```json
{
    "id": true,
    "name": true,
    "description": true
}
```

If applied to the `/skills` endpoint, the above would cause only the "id", "name", and "description" field to be returned. To exclude fields, you would do the following.

```json
{
    "slug": false,
    "ranks": false
}
```

Again, applied to the `/skills` endpoint, the above would return all fields except for the "slug" and "ranks" field.

Unless you have any requests that you'd like to see included with this change, I'm gonna plan on releasing it sometime later today or early tomorrow.

**Comment by TanukiSharp** [comment]
Looks good, just make sure `_id: true` gets rejected if present and should be good to go.
Some tools make the double quotes around properties optional, that would be great if you could support that but definitely a very low priority bonus.

**Comment by LartTyler** [comment]
@TanukiSharp This is now available in [v1.12.0](https://github.com/LartTyler/MHWDB-API/releases/tag/1.12.0). Full usage documentation can be found [here](https://docs.mhw-db.com/#projecting-results).
